README.md

```diff
@@ -1727,6 +1727,29 @@ collectd::plugin::tail::file { 'exim-log':
 }
 ```
 
+#### Class: `collectd::plugin::tail_csv`
+
+```puppet
+class { '::collectd::plugin::tail_csv':
```

Comment:
please don't use the leading ::. This is a puppet 3 relic that isn't needed anymore.